boot.js

```diff
@@ -237,7 +226,10 @@ Boot.prototype._loadRegistered = function () {
 Object.defineProperty(Boot.prototype, 'then', { get: thenify })
 
 Boot.prototype._addPlugin = function (plugin, opts, isAfter) {
-  plugin = assertPlugin(plugin)
+  validatePlugin(plugin)
+  if (isBundledOrTypescriptPlugin(plugin)) {
+    plugin = plugin.default
+  }
```

Comment:
`isBundledOrTypescriptPlugin` seems to be used for the same purpose twice, maybe we can change the order so the outcome is used for `validatePlugin`?


```suggestion
  if (isBundledOrTypescriptPlugin(plugin)) {
    plugin = plugin.default
  }
  validatePlugin(plugin)
```

Unless I'm missing or overlooking, this can remove the dependency for `validatePlugin` to also test wether is bundled or TS import, wdyt?

Reply:
yes thats good.